common/src/main/java/com/tc/logging/TCLoggerProvider.java

```diff
@@ -18,7 +18,9 @@
  */
 package com.tc.logging;
 
+import org.slf4j.Logger;
+
 public interface TCLoggerProvider {
```

Comment:
No this is used. This class has nothing to do with old TCLogging. It's just in the name.